packages/docs/src/scripts/helpers/polyfills.js

```diff
@@ -5,6 +5,7 @@
 // - github.com/facebook/react/issues/8379#issuecomment-264867090
 require('core-js/fn/array/includes');
 require('core-js/fn/array/fill');
+require('core-js/fn/array/find-index');
```

Comment:
Heads up: These polyfills are only loaded on the documentation site, so this will likely need moved into the relevant component file.